lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2376,8 +2376,20 @@ struct ConvertElementwiseOp : ConversionPattern {
     // Add the indexing map for the outs init tensor.
     indexingMaps.push_back(rewriter.getMultiDimIdentityMap(resultRank));
 
+    SmallVector<int64_t> staticResultShape = llvm::to_vector<4>(
+        llvm::map_range(resultShape, [&](Value &v) -> int64_t {
+          if (auto constOp = v.getDefiningOp<arith::ConstantOp>())
+            return constOp.getValue().cast<IntegerAttr>().getInt();
+          return -1;
+        }));
+
+    for (int i = staticResultShape.size() - 1; i >= 0; i--) {
+      if (staticResultShape[i] != -1)
+        resultShape.erase(resultShape.begin() + i);
+    }
+
```

Comment:
question: is there a reason why this is not being handled by `dispatchIndexOpFoldResults` here

https://github.com/llvm/llvm-project/blob/9e6f88b31a7f7957a850d3541ffa759f2993ffc1/mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp#L875-L876

Definition of `dispatchIndexOpFoldResults`: https://github.com/llvm/llvm-project/blob/fe68088d44f760c7d3d8ee6735d396d97cb55478/mlir/lib/Dialect/Utils/StaticValueUtils.cpp#L22